core/templates/core/login.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Logowanie{% endblock %}
+
+
+{% block head %}
+<script type="module">
+    const canvas = document.getElementById('starback-bg');
+          const starback = new Starback(canvas, {
+              type: 'dot',
+              quantity: 250,
+              height: window.innerHeight,
+              width: window.innerWidth,
+              backgroundColor: [],
+              randomOpacity: true,
+          });
+          function resizeCanvas() {
+              canvas.width = window.innerWidth;
+              canvas.height = window.innerHeight;
+          }
+          window.addEventListener('resize', resizeCanvas);
+</script>
+{% endblock %}
+{% block content %}
+<canvas id="starback-bg" class="fixed -z-[1]"></canvas>
+<div class="h-[80vh] flex justify-center flex justify-items-center place-content-center overflow-hidden">
+    <div class="h-[850px] w-screen flex flex-col items-center mt-[60px] text-center">
+        {% load static %}
+        <div class="h-24 w-96 mb-[30px]">
+            <a href="https://discord.com/">
+                <img src="{% static 'discord-logo-blue.svg' %}" alt="Official Discord logo"/>
+            </a>
+        </div>
+        <div class="rounded-lg border-4 shadow-[0_0_2px_#ffff,inset_0_0_2px_#ffff,0_0_5px_#ffff,0_0_15px_#ffff,0_0_30px_#ffff] rounded-lg bg-[#313338] h-3/5 w-1/4 grid justify-items-center">
```

Comment:
Try to improve this shadow if it is possible...

Reply:
Can I keep it in this way anyways?